src/test/scala/ch/epfl/bluebrain/nexus/tests/admin/AdminSpec.scala

```diff
@@ -42,44 +46,51 @@ class AdminSpec extends BaseSpec with OptionValues with CancelAfterFailure with
 
   private def entityMeta(id: String, rev: Long, deprecated: Boolean = false): Json =
     Json.obj(
-      "@context"       -> Json.fromString(config.prefixes.coreContext.toString()),
-      "@id"            -> Json.fromString(s"$adminBase/projects/$id"),
-      "@type"          -> Json.fromString("nxv:Project"),
-      "nxv:rev"        -> Json.fromLong(rev),
-      "nxv:deprecated" -> Json.fromBoolean(deprecated)
+      "@context"    -> Json.fromString(config.prefixes.coreContext.toString()),
+      "@id"         -> Json.fromString(s"$adminBase/projects/$id"),
+      "@type"       -> Json.fromString("nxv:Project"),
+      "_rev"        -> Json.fromLong(rev),
+      "_deprecated" -> Json.fromBoolean(deprecated)
     )
 
   "An Admin service" when {
     val replSub = Map(quote("{sub}") -> config.iam.userSub)
 
-    "managing ACLs" should {
-
-      "delete all ACLs for user" in {
+    def cleanAcls = {
 
-        cl(Req(uri = s"$iamBase/acls/", headers = headersGroup)).mapJson { (json, result) =>
-          result.status shouldEqual StatusCodes.OK
-          val permissions = json.hcursor.get[Vector[Json]]("acl").toOption.value.foldLeft(Set.empty[String]) {
+      cl(Req(uri = s"$iamBase/acls/*?parents=true", headers = headersGroup)).mapJson { (json, result) =>
+        result.status shouldEqual StatusCodes.OK
+        val permissions =
+          json.hcursor.get[Vector[Json]]("acl").toOption.value.foldLeft(Set.empty[(String, Set[String])]) {
             case (acc, j) =>
               j.hcursor.downField("identity").get[String]("sub") match {
                 case Right(s) if s == config.iam.userSub =>
-                  j.hcursor.get[Set[String]]("permissions").toOption.value ++ acc
+                  acc + Tuple2(j.hcursor.get[String]("path").toOption.value,
```

Comment:
Is the Tuple2 needed? shouldn't `a -> b` or (a, b) should be enough?

Reply:
for some reason scala compiler didn't like `(a, b)`, but `(a -> b)` seems to work, will change that.